snowfakery/data_generator_runtime.py

```diff
@@ -360,12 +357,17 @@ def __init__(
 
         self.templates = templates
 
-    def loop_over_templates_until_finished(self, runtimecontext, continuing):
+    # TODO: move this into the interpreter object
+    def loop_over_templates_until_finished(self, continuing):
         finished = False
+        # TODO: make a fresh runtime context per execution to avoid
+        #       risk of data leakage
+        runtimecontext = RuntimeContext(interpreter=self)
```

Comment:
Looks like these todos were done?

Reply:
Removed in 0730497

Reply:
@prescod I meant "# TODO: move this into the interpreter object" too

Reply:
Makes sense. 0234afd